shared/js/ui/templates/privacy-options.es6.js

```diff
@@ -36,5 +36,19 @@ module.exports = function () {
             </p>
         </li>
     </ul>
+    <!--
```

Comment:
Can we delete this? It's comments only.

Reply:
I suggested we comment it, we'll uncomment with the next PR when we enable the feature.

Reply:
Yes, the idea is to keep it since this setting is ready, we're just not ready to enable it yet while we don't launch the feature, if that's ok.